tests/e2e/api/collection.test.ts

```diff
@@ -1,44 +1,54 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-// import { Aptos, AptosConfig, Network } from "../../../src";
+import { Account, Aptos, AptosConfig, Network } from "../../../src";
+import { waitForTransaction } from "../../../src/internal/transaction";
+import { FUND_AMOUNT } from "../../unit/helper";
+
+// use it here since all tests use the same configuration
+const config = new AptosConfig({ network: Network.DEVNET });
```

Comment:
I believe we will want to change these to LOCAL, but let's evaluate that at a combined level